BaseStation/PTZCamera/Camera_Control.cs

```diff
@@ -0,0 +1,81 @@
+﻿using OpenTK;
+using OpenTK.Input;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HuskyRobotics
+{
+    /// <summary>
+    /// This class is just used to test the camera controls.
+    /// </summary>
+    class Camera_Control
+    {
+        static void Main(string[] args)
+        {
+            int max_speed = 100;
+            int x_speed=0;
+            int y_speed=0;
+            bool speed_Changed;
+            HttpClient client = new HttpClient();
+            PTZCamera cam = new PTZCamera("192.168.0.42", "admin","1234",client);
```

Comment:
Should this be configurable?

Reply:
Nope! We don't plan to change the IP of the camera, may be nice in future but it is a static ip device.